src/core/client/ui/components/AbsoluteTime/AbsoluteTime.tsx

```diff
@@ -1,16 +1,19 @@
 import React, { FunctionComponent, useMemo } from "react";
 
-import { useCoralContext } from "coral-framework/lib/bootstrap";
 import { Typography } from "coral-ui/components";
 import { PropTypesOf } from "coral-ui/types";
 
 interface Props {
   date: string;
   className?: string;
+  locales: string[];
 }
 
-const AbsoluteTime: FunctionComponent<Props> = ({ date, className }) => {
-  const { locales } = useCoralContext();
+const AbsoluteTime: FunctionComponent<Props> = ({
+  date,
+  className,
+  locales,
+}) => {
   const formatted = useMemo(() => {
```

Comment:
What do you think of this way?
```ts
const { locales: localesFromContext } = useUIContext();
const locales = localesFromProps || localesFromContext || ["en-US"];
```
`useUIContext()` doesn't exist but you can easily construct it, just look at how `useCoralContext` is created.